### ISSUE TITLE
Unable to find 'Debian.yml' in expected paths

### ISSUE THREAD

**Issue by quanticware** [PowerDNS/pdns_recursor-ansible#23]
Hi,

Debian.yml is absent from your tasks directory

```
TASK [pdns_recursor-ansible : Include OS-specific variables] *******************
task path: /etc/ansible/roles/pdns_recursor-ansible/tasks/main.yml:3
fatal: [host-02]: FAILED! => {
    "ansible_facts": {}, 
    "changed": false, 
    "failed": true, 
    "invocation": {
        "module_args": {
            "_raw_params": "Debian.yml"
        }, 
        "module_name": "include_vars"
    }, 
    "message": "Unable to find 'Debian.yml' in expected paths."
}
        to retry, use: --limit @/home/ubuntu/ansible/play/ex-pdns-recursor.retry
```

I had to comment those two lines in the main.yml task file :

```
#- name: Include OS-specific variables
#  include_vars: "{{ ansible_os_family }}.yml"
```

My mistake ?
Cheers

**Comment by atosatto** [comment]
Ehi @quanticware thank you for reporting this issue.

A better workaround would be creating an empty `vars/Debian.yml` file or using the branch `fix-#23` (see PR #24) including the fix for you issue.

**Comment by atosatto** [comment]
The fix has now been merged. @quanticware can you please give another chance to master?!
